### PR TITLE
fix: update message for registered user response

### PR DESCRIPTION
**Pull request by dvandra**
Update message for registered user response:

- Do not show exploit details if exploitable vulnerability count is 0.

- Show highest severity of the vulnerability.

- Also show recommended version for only security advisories. (for registered user only)

https://issues.redhat.com/browse/APPAI-1479 
https://issues.redhat.com/browse/APPAI-1512

### CODE REVIEW

File: bayesian/utility/v2/ca_response_builder.py [hunk 1]
```diff
@@ -472,24 +472,35 @@ def get_premium_message(self, exploitable_vuls: int) -> str:
...
+            message += f"{self.pvt_vul} security advisory with "
+            message += self.append_with_exploit_details(exploitable_vuls)
+            message += f"{self.severity[0]} severity. "
+            message += self.get_recommendation()
         return message
 
+    def append_with_exploit_details(self, exploitable_vuls) -> str:
+        """Append exploit details to message."""
+        if exploitable_vuls:
+            message = f"{exploitable_vuls} exploitable vulnerability and "
```
Comment (dgpatelgit) [inline bayesian/utility/v2/ca_response_builder.py:500]: Ignore 'message' variable.

File: tests/utility/v2/test_ca_response_builder.py [hunk 2]
```diff
@@ -197,8 +198,10 @@ def test_get_premium_message_with_pvt_vul(self):
...
         obj.package = "django"
         obj.version = "1.1"
+        obj.severity = ['high']
         msg = obj.get_premium_message(1)
-        ideal_msg = "django - 1.1 has 1 security advisory with 1 exploitable vulnerabilities. "
+        ideal_msg = "django - 1.1 has 1 security advisory with 1 exploitable " \
+                    "vulnerability and high severity. No recommended version."
         self.assertEqual(msg, ideal_msg)
 
     def test_get_premium_message_with_public_vul(self):
```
Comment (dgpatelgit) [inline tests/utility/v2/test_ca_response_builder.py:207]: Add test case for '0' exploit and 'n' exploit.

Comment (dvandra) [reply tests/utility/v2/test_ca_response_builder.py:207]: Done.

File: bayesian/utility/v2/ca_response_builder.py [hunk 3]
```diff
@@ -472,24 +472,34 @@ def get_premium_message(self, exploitable_vuls: int) -> str:
...
             message += self.get_recommendation()
             return message
 
         if self.public_vul:
             # Add Public Vulnerability Info only
-            message += f"{self.public_vul} known security vulnerability"
-            message += f" with {exploitable_vuls} exploitable vulnerabilities. "
+            message += f"{self.public_vul} known security vulnerability with "
+            message += self.append_with_exploit_details(exploitable_vuls)
+            message += f"{self.severity[0]} severity. "
```
Comment (dgpatelgit) [inline bayesian/utility/v2/ca_response_builder.py:485]: Is this 'high severity' text is extra in new message, it was not there before your changes?

File: bayesian/utility/v2/ca_response_builder.py [hunk 4]
```diff
@@ -472,24 +472,34 @@ def get_premium_message(self, exploitable_vuls: int) -> str:
         if self.public_vul and self.pvt_vul:
             # Add Private Vulnerability and Public Vul Info only
             message += f"{self.public_vul} known security vulnerability "
-            message += f"and {self.pvt_vul} security advisory "
-            message += f"with {exploitable_vuls} exploitable vulnerabilities. "
+            message += f"and {self.pvt_vul} security advisory with "
+            message += self.append_with_exploit_details(exploitable_vuls)
+            message += f"{self.severity[0]} severity. "
```
Comment (dgpatelgit) [inline bayesian/utility/v2/ca_response_builder.py:477]: Severity is added now?

File: bayesian/utility/v2/ca_response_builder.py (hunk 3 above)
Comment (dvandra) [reply bayesian/utility/v2/ca_response_builder.py:485]: Yes, it is extra and it should be there: 
Jira story: https://issues.redhat.com/browse/APPAI-1512

File: bayesian/utility/v2/ca_response_builder.py (hunk 4 above)
Comment (dvandra) [reply bayesian/utility/v2/ca_response_builder.py:477]: Yes, it is added now.

**Review by dgpatelgit — APPROVED** [review]
lgtm

**Pull request merged**